contrib/kserve/models-web-app/base/istio.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   http:
   - match:
     - uri:
-        prefix: /models/
+        prefix: /kserve-endpoints/
```

Comment:
Does this mean that the [notebook configmap](https://github.com/kubeflow/manifests/blob/master/apps/centraldashboard/upstream/base/configmap.yaml#L30) should change as well? I was able to successfully test out standalone model web app, but with the full kubeflow installation, I am just getting a blank page for models page which points to `http://localhost:8080/_/kserve-endpoints/?ns=kubeflow-user-example-com`

Reply:
Can you confirm you are also using the `kserve` overlay when deploying the CentralDashboard? We had introduced this in the previous release to support both KServe and KFServing.

https://github.com/kubeflow/manifests/blob/v1.6-branch/example/kustomization.yaml#L33

Reply:
Thanks for the clarification Kimonas! I retested with your branch and got it up and running successfully with full installation! Sorry for the confusion!